shared/util.go

```diff
@@ -1358,3 +1358,35 @@ func TargetDetect(target string) (targetNode string, targetGroup string) {
 
 	return
 }
```

Comment:
Just a general comment for the commit message. We tend to use the Go package name as the prefix so in this case it would be something like `lxd/shared: ...`.